### PR TITLE
feat(icon): DLT-2145 added Workforce Management (née Surfboard)

### PR DESCRIPTION
**Pull request by fede-dp**
# DLT-2145 new icon WFM v2

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will increment the version number on release:

- [ ] Fix
- [x] Feature
- [ ] Performance Improvement
- [ ] Refactor

These types will not increment the version number, but will still deploy to documentation site on release:

- [ ] Documentation
- [ ] Build system
- [ ] CI
- [ ] Style (code style changes, not css changes)
- [ ] Test
- [ ] Other (chore)

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [x] I have added all relevant documentation.
- [x] I have considered the performance impact of my change.

## :camera: Screenshots / GIFs

image

### CODE REVIEW

File: packages/dialtone-icons/src/keywords-icons.json [hunk 1]
```diff
@@ -1549,6 +1549,13 @@
         "payment",
         "cc"
       ],
+      "dialpad-WFM": [
+        "workforce",
+        "surfboard",
+        "work",
+        "force",
```
Comment (francisrupert) [inline packages/dialtone-icons/src/keywords-icons.json]: "work" and "force" are redundant. Remove.

File: packages/dialtone-icons/src/keywords-icons.json [hunk 2]
```diff
@@ -1549,6 +1549,13 @@
         "payment",
         "cc"
       ],
+      "dialpad-WFM": [
```
Comment (francisrupert) [inline packages/dialtone-icons/src/keywords-icons.json]:
```suggestion
      "dialpad-wfm": [
```
Lowercase

**Review by francisrupert — APPROVED** [review]
Approved, though with the changes below.

**Review by iropolo — APPROVED** [review]

**Comment by francisrupert** [comment]
@fede-dp While you're still in here can you move `dialpad-sparkle` from `brand` to `brand-full-color` category.

image

**Pull request merged**